runtime/lua/nvim/util.lua

```diff
@@ -0,0 +1,113 @@
+local util = {
+  quickfix = {},
+}
+
+util.tostring = function(obj)
+  local stringified = ''
+  if type(obj) == 'table' then
+    stringified = stringified .. '{'
+    for k, v in pairs(obj) do
+      stringified = stringified .. util.tostring(k) .. '=' .. util.tostring(v) .. ','
+    end
+    stringified = stringified .. '}'
+  else
+    stringified = tostring(obj)
+  end
+
+  return stringified
+end
+
+util.handle_uri = function(uri)
+  local file_prefix = 'file://'
+  if string.sub(uri, 1, #file_prefix) == file_prefix then
+    return string.sub(uri, #file_prefix + 1, #uri)
+  end
```

Comment:
this is a rather special-purpose function that's only used in 1 place. Wouldn't it make more sense to have something like `uri_scheme()` which returns "http" for "http://foo.com" ?

Reply:
Thanks. I think windows can't use this function. So I rewrite this after fixing LSP api.:)